FirebaseDatabase/Sources/Api/FIRDatabase.m

```diff
@@ -173,6 +192,26 @@ - (void)purgeOutstandingWrites {
     });
 }
 
+- (void)useEmulatorWithHost:(NSString *)host port:(NSInteger)port {
+    if (host.length == 0) {
+        [NSException raise:NSInvalidArgumentException
+                    format:@"Cannot connect to nil or empty host."];
+    }
+    if (self.repo != nil) {
+        [NSException raise:NSInternalInconsistencyException
```

Comment:
Do you mind re-wrapping this to match the longer line length in this file?

Reply:
Done.